lib/pure/distros.nim

```diff
@@ -187,14 +192,17 @@ proc detectOsImpl(d: Distribution): bool =
         # Check if this is a Nix build or NixOS environment
         result = existsEnv("NIX_BUILD_TOP") or existsEnv("__NIXOS_SET_ENVIRONMENT_DONE")
       of Distribution.OpenSUSE:
-        result = "suse" in toLowerAscii(uname()) or "suse" in toLowerAscii(release())
+        result = "suse" in toLowerAscii(uname()) or "suse" in toLowerAscii(
+            release())
```

Comment:
It's not really worth it to start a new line here.

Reply:
Same as above. I'll reformat it back.